### ISSUE TITLE
Game crashes if JEI is not installed

### ISSUE THREAD

**Issue by mikkolukas** [hlysine/create_connected#8]
Just Enough Items (JEI) is listed as an optional dependency, but Minecraft crashes on load if JEI is not installed.

If JEI is installed, Minecraft loads to the main menu as expected.

---

Crash report pasted here: [https://mclo.gs/N2k18yp](https://mclo.gs/N2k18yp)

```
Minecraft Version: 1.20.1
Minecraft Version ID: 1.20.1
Operating System: Windows 11 (amd64) version 10.0
Java Version: 17.0.8.1, Eclipse Adoptium
```

**Comment by hlysine** [comment]
Thanks for the report. It has been fixed in #7 and will be released in the next update very soon.

**Comment by mikkolukas** [comment]
> Thanks for the report. It has been fixed in #7 and will be released in the next update very soon.

Ah, when opening issues, it only shows open issues by default. I didn't get the idea to look at the closed issues 🤦

I don't know if it is possible or even a good idea, but it could be beneficial to let the issues stay open until the fix is released.

Thanks for a great mod! 🙂👍

**Comment by hlysine** [comment]
In general, it is a good idea to search through all issues, open or closed, before creating a new one, but don't worry, I have nothing against duplicate issues.

I think it's a common practice to close issues once the fix is pushed, mainly because it is more convenient to write "fixes #X" and let GitHub close issues automatically.
